.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: required
 
+before_install:
+  - sudo apt-get update
```

Comment:
If you see the CI log it says
```
ERROR: Service 'web' failed to build: The command '/bin/sh -c apt-get update -qq && apt-get install -y bundler libmysqlclient-dev ruby-rmagick libfreeimage3 nodejs-legacy npm wget openjdk-7-jre openjdk-7-jdk' returned a non-zero code: 100
```
Actually some packages didn't install properly. In this case I hope just closing and reopening the pull request would do.